datasets/pl/wanted/crawler.py

```diff
@@ -0,0 +1,160 @@
+    ), "Could not extract r-ID from malformed person URL"
+    rid = url_match.groupdict()["rid"]
+    person.id = context.make_slug(rid)
+
+    doc = context.fetch_html(url, cache_days=7)
+    # Extract details using XPath based on the provided HTML structure
+    details = {
+        "full_name": "//div[@class='head']/h2/text()",
+        "middle_name": "//p[contains(text(), 'Data urodzenia:')]/strong/text()",
+        "father_name": "//p[contains(text(), 'Imię ojca:')]/strong/text()",
+        "mother_name": "//p[contains(text(), 'Imię matki:')]/strong/text()",
+        "mother_maiden_name": "//p[contains(text(), 'Nazwisko panieńskie matki:')]/strong/text()",
+        "gender": "//p[contains(text(), 'Płeć:')]/strong/text()",
+        "place_of_birth": "//p[contains(text(), 'Miejsce urodzenia:')]/strong/text()",
+        "date_of_birth": "//p[contains(text(), 'Data urodzenia:')]/strong/text()",
+        "alias": "//p[contains(text(), 'Pseudonim:')]/strong/text()",
+        "citizenship": "//p[contains(text(), 'Obywatelstwo:')]/strong/text()",
+        "height": "//p[contains(text(), 'Wzrost:')]/strong/text()",
+        "eye_color": "//p[contains(text(), 'Kolor oczu:')]/strong/text()",
+    }
+    info = dict()
+    for key, xpath in details.items():
+        q = doc.xpath(xpath)
+        if q:
+            text = q[0].strip()
+            if text != "-":
+                info[key] = text
+
+    person.add("sourceUrl", url)
+    person.add("topics", "crime")
+    person.add("topics", "wanted")
+    # TODO(Leon Handreke): What does country mean in that context? The za_wanted crawler also does this.
+    person.add("country", "pl")
+
+    h.apply_name(person, full=info["full_name"], middle_name=info.get("middle_name"))
+    h.apply_date(person, "birthDate", info.get("date_of_birth"))
+    person.add("birthPlace", info.get("place_of_birth"))
+    person.add("gender", info.get("gender"))
+    person.add("alias", info.get("alias"))
+    person.add("fatherName", info.get("father_name"))
+    person.add("motherName", info.get("mother_name"))
+    # TODO(Leon Handreke): Add mother maiden name or is that too detailed?
+    person.add("motherName", info.get("mother_maiden_name"))
+    person.add("height", info.get("height"))
+    person.add("eyeColor", info.get("eye_color"))
+
+    citizenship = info.get("citizenship")
+    cleaned_citizenship = (
+        clean_and_split_citizenship(citizenship) if citizenship else None
+    )
+
+    person.add("citizenship", cleaned_citizenship, original_value=citizenship)
+
+    crimes = doc.xpath(
+        "//p[contains(text(), 'Podstawy poszukiwań:')]/following-sibling::ul//a/text()"
+    )
+    if not crimes:
+        context.log.warn("No crimes found for person", entity_id=person.id, url=url)
+    # TODO(Leon Handreke): Every wanted list puts this in notes -- could there be a better place?
+    person.add("notes", crimes)
+
+    context.emit(person, target=True)
```

Comment:
no need for `target` any more. We [recently](https://www.opensanctions.org/changelog/13/) switched to setting `Entity.target` based on https://github.com/opensanctions/opensanctions/blob/main/zavod/zavod/settings.py#L33

Soon we should do a cleanup.

```suggestion
    context.emit(person)
```

Reply:
This is the sort of thing that IntelliJ should be good at :)